pkg/pool-stable/test/foundry/StablePool.t.sol

```diff
@@ -172,6 +172,16 @@ contract StablePoolTest is BasePoolTest, StablePoolContractsDeployer {
         assertTrue(isUpdating, "Amplification update by Bob not started");
     }
 
+    function testStartAmplificationParameterUpdateIsPermissioned() public {
+        vm.expectRevert(IAuthentication.SenderNotAllowed.selector);
+        IStablePool(pool).startAmplificationParameterUpdate(DEFAULT_AMP_FACTOR * 2, block.timestamp + 5000 days);
+    }
+
+    function testStopAmplificationParameterUpdateIsPermissioned() public {
+        vm.expectRevert(IAuthentication.SenderNotAllowed.selector);
+        IStablePool(pool).stopAmplificationParameterUpdate();
+    }
+
```

Comment:
We should add a test where the swap fee manager performs the permissioned functions

Also maybe show that when there's a swap fee manager, the usual governance permission doesn't work